release/models/aft/openconfig-aft-common.yang

```diff
@@ -22,7 +22,15 @@ submodule openconfig-aft-common {
     "Submodule containing definitions of groupings that are re-used
     across multiple contexts within the AFT model.";
 
-  oc-ext:openconfig-version "0.9.0";
+  oc-ext:openconfig-version "1.0.0";
```

Comment:
Do we want to cut `1.0.0` here -- it seems reasonable, but maybe needs more consideration -- or just do `0.10.0`?

Reply:
Since `1.0.0` has since been published (and a few other PRs are in flight touching this domain) - this PR is now updated to reflect `2.0.0`.  This is a backwards incompatible change thus the major rev should be incremented here